.github/actions/clang-format/entrypoint.sh

```diff
@@ -11,6 +11,7 @@ echo "Reverting any suggestions made to generated files"
 find . -name "*.pb-c.c" -exec git checkout --quiet ./'{}' \;
 find . -name "*.pb-c.h" -exec git checkout --quiet ./'{}' \;
 find . -name "*.proto" -exec git checkout --quiet ./'{}' \;
+find . -name "*.json" -exec git checkout --quiet ./'{}' \;
```

Comment:
This should probably have read `find src/control/system/raft/testdata -name "*.json" -exec git checkout --quiet ./'{}' \;` as there is other json in tree we would like to keep will formatted.